src/Core/Domain/SqlManagement/ValueObject/SqlRequestId.php

```diff
@@ -45,7 +45,7 @@ class SqlRequestId
      */
     public function __construct($requestSqlId)
```

Comment:
I was thinking about casting string parameters into integers:

```suggestion
    public function __construct(int $requestSqlId)
```

It might not be the best solution though, it will break into a fatal error if the parameter is not a numeric value.

You should probably catch where is this ValueObject given a string value...

Reply:
Fixed in controller.